landlab/plot/landslides/plot_landslides.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+"""plot_landslides.py
+Created on Mon May 9 2020
+@author: christinab
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from landlab.plot import imshow_grid
+
+
+def test_field_initialized_to_range(ls_prob, input_var_names, unit_default_value):
+    """Testing if the output fields are initialized within range of default.
+    """
+    for name in input_var_names:
+
+        print(
+            "Input: (Min, Max) range of {value} model inputs: {pars}".format(
+                value=name, pars=unit_default_value[name]
+            )
+        )
+        print(
+            "Ouput: {value} default value initialized for each node:".format(value=name)
+        )
+        field = ls_prob.grid["node"][name]
+        print(field)
+
+        field_min = ls_prob.grid["node"][name].min()
+        field_max = ls_prob.grid["node"][name].max()
+        unit_min = unit_default_value[name][0]
+        unit_max = unit_default_value[name][1]
+
+        assert field_min >= unit_min
+        assert field_max <= unit_max
+
+        print("Test: {value} within range. Passes test.".format(value=name))
+        print("")
+
+
+def print_nodevalues(ls_prob):
+    """Print out the input range and resulting node values for 
```

Comment:
This seems like an odd function to be located in the plot_landslides.py file. Recommend putting it somewhere more appropriate. Perhaps @mcflugen has a recommendation as well.